secure-biometric-device/README.md

```diff
@@ -0,0 +1,145 @@
+# Secure Biometric Device
+
+A standalone vanillajs library component for interacting with SBI devices and auth capture Face, Finger & Iris detail.
+
+## Build
+
+To build up the library
+
+```bash
+npm run build
+```
+
+To generate a package from it
+
+```bash
+npm pack
+```
+
+For publishing the package
+
+```bash
+npm publish
+```
+
+## Local
+
+### In Html
+
+Either you build the library or serve the libray
+
+```bash
+npm run build
+
+or 
+
+npm run start
+```
+
+After that you can serve the index.html file from the below folder
+```
+secure-biometric-device
+└─── example
+     └─── html
+          │─── index.html
+          └─── mbd.js
+```
+
+How are we implement the library in vanilla javascript, checkout the `mbd.js` file (see above path), below you can see the snippet also.
+
+```js
+SecureBiometricDevice.init({
+  container: document.getElementById("secure-biometric-device"),
+  biometricEnv: BIOMETRIC_ENV_OBJECT,
+  buttonLabel: BUTTON_LABEL,
+  disable: DISABLE_OR_NOT,
+  transactionId: TRANSACTION_ID,
+  onCapture: SUCCESS_CALLBACK_METHOD,
+  onErrored: ERROR_CALLBACK_METHOD
+})
+```
+
+### In React
+
+For running our library in react, first of all build the secure-biometric-device library and bundle up as a pakcage
+
+```bash
+npm run package
+```
+
+After that you can go to our react example of the project, install the node_modules and run as a react app
+```
+secure-biometric-device
+└─── example
+     └─── react
+          │─── public
+          │─── src
+          |    │─── App.js
+          |    └─── index.js
+          └─── package.json
+```
+
+Code snippet of using secure-biometric-device library in react is given below ( you can also find it in `App.js` file of src folder)
+
+```js
+// import the library first
+import { init } from "secure-biometric-device";
+
+// call the library method
+init({
+    container: document.getElementById("secure-biometric-device"),
```

Comment:
Can we rename this to secure-biometric-interface-integration. @anshulv1401 can you confirm on this ?

Reply:
@vishwa-vyom , yes, it can be renamed in the doc. We are expecting the ID to be defined and passed by the implementor.

Reply:
hi @vishwa-vyom , this has been done, it can be checked in [b321370](https://github.com/mosip/esignet-plugins/pull/5/commits/b321370a71e6d926522bb0f944139bf13c4b04c2) commit

Reply:
it should be secure-biometric-interface-integration and not integrator for the div

Reply:
hi @vishwa-vyom , this is also done, with [975a8fe](https://github.com/mosip/esignet-plugins/pull/5/commits/975a8fe0796cdf47b1c5d5b77973c3b90988be89) commit